server/src/main/java/org/elasticsearch/index/mapper/SeqNoFieldMapper.java

```diff
@@ -95,23 +81,21 @@ public String toString() {
      */
     public static class SequenceIDFields {
 
-        private final Field seqNo;
-        private final Field primaryTerm;
-        private final Field tombstoneField;
+        private static final Field TOMBSTONE_FIELD = new NumericDocValuesField(TOMBSTONE_NAME, 1);
+
+        private final Field seqNo = new SingleValueLongField(NAME);
+        private final Field primaryTerm = new NumericDocValuesField(PRIMARY_TERM_NAME, 0);
+        private final boolean isTombstone;
```

Comment:
@original-brownbear why is `new SingleValueLongField(NAME);` this and not `new SingleValueLongField(NAME, SequenceNumbers.UNASSIGNED_SEQ_NO),` like it was before?